docroot/modules/humsci/hs_paragraph_types/modules/hs_hero_image/config/install/responsive_image.styles.hero_with_text.yml

```diff
@@ -0,0 +1,29 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - image.style.hero_with_text_large
+    - image.style.responsive_medium
+    - image.style.responsive_small
+  module:
+    - stanford_media
+id: hero_with_text
+label: 'Hero with Text'
+image_style_mappings:
```

Comment:
Did you forget the 2x pixel density screens?

Reply:
same as above :point_up: